apps/services/user-notification/src/app/modules/notifications/notifications.module.ts

```diff
@@ -84,26 +89,35 @@ import {
               ),
             }),
     },
-    {
-      provide: userProfile.UserProfileApi,
+    ...[
+      {
+        api: V2UsersApi,
+        name: 'services-user-notification-v2',
+      },
+      {
+        api: V2UserTokensApi,
+        name: 'services-user-token-profile-v2',
+      },
+    ].map(({ api, name }) => ({
+      provide: api,
       useFactory: () =>
-        new userProfile.UserProfileApi(
-          new userProfile.Configuration({
+        new api(
+          new Configuration({
             basePath: environment.userProfileServiceBasePath,
             fetchApi: createEnhancedFetch({
-              name: 'services-user-notification',
+              name: name,
```

Comment:
When this is refactored to the client library the convention is to use the same name for all apis provided. In this case it should be `clients-user-profile`